### ISSUE TITLE
WhitespaceAround: allowEmptyTypes skip validation for non empty class

### ISSUE THREAD

**Issue by romani** [checkstyle/checkstyle#2480]
```
$ java -jar checkstyle-6.11.2-all.jar -c checkstyle.xml test.java
Starting audit...
Audit done.

/var/tmp $ cat test.java 
public class CheckstyleTest{ // Doesn't warn - incorrect
private static final int SOMETHING = 1;
}

/var/tmp $ cat checkstyle.xml 
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
		"-//Puppy Crawl//DTD Check Configuration 1.3//EN"
		"http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
<module name="Checker">
	<module name="TreeWalker">
		<module name="WhitespaceAround">
			<property name="allowEmptyTypes" value="true"/>
		</module>
	</module>
</module>
```

Expected: violation on first line.
```/var/tmp/test.java:1:28: error: '{' is not preceded with whitespace.```

**Comment by romani** [comment]
merged.
